modules/accumulo/src/main/java/org/apache/fluo/recipes/accumulo/export/AccumuloExporter.java

```diff
@@ -17,27 +17,55 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.HashMap;
 import java.util.Iterator;
+import java.util.Map;
 
 import org.apache.accumulo.core.data.Mutation;
-import org.apache.fluo.api.observer.Observer.Context;
+import org.apache.fluo.api.config.SimpleConfiguration;
+import org.apache.fluo.api.data.Bytes;
+import org.apache.fluo.api.data.Column;
+import org.apache.fluo.api.data.RowColumn;
+import org.apache.fluo.recipes.core.export.ExportQueue;
 import org.apache.fluo.recipes.core.export.Exporter;
 import org.apache.fluo.recipes.core.export.SequencedExport;
 
 /**
- * An Accumulo-specific {@link Exporter} that writes mutations to Accumulo using a
- * {@link AccumuloExportQueue.AccumuloWriter}
+ * An Accumulo-specific {@link Exporter} that writes mutations to Accumulo. For an overview of how
+ * to use this, see the project level documentation for exporting to Accumulo.
  *
  * @since 1.0.0
  */
 public abstract class AccumuloExporter<K, V> extends Exporter<K, V> {
 
-  private AccumuloExportQueue.AccumuloWriter accumuloWriter;
+  /**
+   * Use this to configure the Accumulo table where an AccumuloExporter's mutations will be written.
+   * Create and pass to {@link ExportQueue.Options#setExporterConfiguration(SimpleConfiguration)}
+   *
+   * @since 1.0.0
+   */
+  public static class Configuration extends SimpleConfiguration {
+    String instanceName;
```

Comment:
These fields are now not used as constructor sets properties in configuration.  They should be removed.